### ISSUE TITLE
Training Hanging for Binary and Multi Class classification models

### ISSUE THREAD

**Issue by anaconda121** [ThilinaRajapakse/simpletransformers#1252]
**Describe the bug**
A clear and concise description of what the bug is. Please specify the class causing the issue.
Model Training appears to be hanging. I'm using the classification model for multi-class classification. 

**To Reproduce**
1. https://github.com/ThilinaRajapakse/simpletransformers/blob/master/examples/text_classification/multiclass_classification.py -> Download code
2. Run code.
3. Observe.

**Expected behavior**
What I thought would happen is that the training progress bar would be full.

**Screenshots**
![image](https://user-images.githubusercontent.com/38384138/131930569-144ec2ea-dcbb-4fb3-88ab-558874ab7ce2.png)

**Desktop (please complete the following information):**
 - OS : Windows 10. Also encounter same results as in screenshot when I run the code on google colab.

**Additional context**
Not sure if this is expected as the model appears to be evaluating on the validation set and going through epochs. However, the training is hanging for me on another model (cannot share code b/c it is a corporate project with sensitive data) so would like to know how to address this. 

Let me know if I need to include any other information.
